griptape/tools/google_docs/tool.py

```diff
@@ -244,49 +244,6 @@ def save_memory_artifacts_to_google_docs(self, params: dict) -> ErrorArtifact |
         else:
             return ErrorArtifact("memory not found")
 
-    @activity(
-        config={
-            "description": "Can be used to download multiple Google Docs based on their paths.",
-            "schema": Schema(
-                {
-                    Literal(
-                        "file_paths",
-                        description="Destination file paths of Google Docs in the POSIX format. "
-                        "For example, 'foo/bar/baz.txt, foo/bar/baz2.txt'",
-                    ): list[str]
-                }
-            ),
-        }
-    )
-    def download_google_docs(self, params: dict) -> ListArtifact | ErrorArtifact:
```

Comment:
Did you mean to add this to the Drive tool?

Reply:
I meant, the download function in drive also does this.